tests/canary_upgrade_test.go

```diff
@@ -25,14 +25,16 @@ import (
 	"sync"
 	"time"
 
+	"kubevirt.io/kubevirt/pkg/apimachinery/patch"
```

Comment:
This should be placed in the `kubevirt.io/kubevirt` import section below

Reply:
done